is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
```

Comment:
When the loop reaches the last element of the array, you will get `null` from .slice(currentIndex + 1).  
A control flow might be needed before line 6 to make sure the 2nd .foreach loop with not be triggered.